client/coral-embed-stream/src/CommentStream.js

```diff
@@ -46,7 +51,8 @@ const mapDispatchToProps = (dispatch) => ({
   },
   appendItemArray: (item, property, value, addToFront, itemType) =>
     dispatch(appendItemArray(item, property, value, addToFront, itemType)),
-  logout: () => dispatch(logout())
+  logout: () => dispatch(logout()),
+  changeTab: activeTab => dispatch(changeTab(activeTab))
```

Comment:
yeah, I was just confused. I think this is a better way to do it than how we did it in cay.